src/audio_core/stream.h

```diff
@@ -94,6 +100,7 @@ class Stream {
 
     u32 sample_rate;                          ///< Sample rate of the stream
     Format format;                            ///< Format of the stream
+    float game_volume = 1.0f;                 ///< The volume the game currently has set
```

Comment:
As lioncash reviewed:
```suggestion
    float volume = 1.0f;                 ///< The volume the game currently has set
```